sources/packages/backend/apps/api/src/route-controllers/overaward/_tests_/e2e/overaward.aest.controller.addManualOveraward.e2e-spec.ts

```diff
@@ -40,6 +41,15 @@ describe("OverawardAESTController(e2e)-addManualOveraward", () => {
         BEARER_AUTH_TYPE,
       )
       .expect(HttpStatus.CREATED);
+
+    const updatedStudent = await studentRepo.findOne({
```

Comment:
I would recommend the assertion to be done in this way.

Read the response of post and get the id of overaward generated.

```
const overawrdCreated = overawardRepo.findOne({
select: {id:true, notes: {noteType:true, description:true}},
relations: { notes: true },
 where: { id: overawardId },
}}

expect(overawrdCreated.note.noteType).toBe(NoteType.Overaward);
    expect(overawrdCreated.note.description).toBe(manualOverawardPayload.overawardNotes);
```

Advantage - If the API fails to associate the student note created with the overaward itself. It will be caught by the test.